doctr/datasets/generator/pytorch.py

```diff
@@ -51,4 +54,15 @@ class WordGenerator(_WordGenerator):
         sample_transforms: composable transformations that will be applied to both the image and the target
     """
 
-    pass
+    def __init__(self, *args, **kwargs) -> None:
+        super().__init__(*args, **kwargs)
+        if self.img_transforms is None:
+            img_transforms = Compose([
+                T.RandomApply(RandomRotation(
+                    [-6, 6], interpolation=InterpolationMode.BILINEAR, expand=True), .2),
+                T.Resize((32, 128), preserve_aspect_ratio=True),
+                T.RandomApply(T.ColorInversion(min_val=1.0), 1.0),
+                T.RandomApply(GaussianBlur(
+                    kernel_size=3, sigma=(0.3, 2.0)), .2),
+            ])
+            setattr(self, 'img_transforms', img_transforms)
```

Comment:
A dataset should not have any transforms by default. This is typically the type of operations to define in the training script :+1: